pandas/core/groupby.py

```diff
@@ -191,6 +191,63 @@
     dtype: int64
     """)
 
+_numeric_operations_doc_template = """
+    Compute %(f)s of group values.
+
+    For multiple groupings, the result index will be a
+    :class:`~pandas.MultiIndex`.
+
+    Parameters
+    ----------
+    kwargs : dict
+        Optional keyword arguments to pass to `%(f)s`.
+
+    Returns
+    -------
+    Series or DataFrame
+
+    {examples}
+    {see_also}
+    """
+
+_numeric_operations_examples = dict(
+    max="""Examples
```

Comment:
Reviewers: If you think this design pattern is okay, please let me know if I can add examples for the other generic numeric operators in this module (i.e. `sum`, `prod`, `min`, `first`, `last`) in the same PR

Reply:
to make this easier to edit, create the ``_numeric_operations_examples = {}``
then each entry like
```
_numeric_operations['max'] = dedent(
.....
)
```